hazelcast/src/test/java/com/hazelcast/spi/discovery/multicast/MemberToMemberDiscoveryTest.java

```diff
@@ -44,6 +44,10 @@
     @Rule
     public final OverridePropertyRule overrideJoinWaitSecondsRule = set("hazelcast.wait.seconds.before.join", "10");
     @Rule
+    public final OverridePropertyRule overrideMergeFirstRunDelayRule = set("hazelcast.merge.first.run.delay.seconds", "30");
```

Comment:
You can lower it even further. See our [split brain tests](https://github.com/hazelcast/hazelcast/blob/92ca85687ad1f9cc2057a7cd3b7a27f22bad8019/hazelcast/src/test/java/com/hazelcast/test/SplitBrainTestSupport.java#L156-L160).

Reply:
:+1:  
Done